mldsa/fips202/fips202.c

```diff
@@ -498,14 +498,18 @@ __contract__(
   ensures(return_value <= r))
 {
   unsigned int i;
-  uint64_t lane;
-  uint8_t byte;
+  size_t out_offset = 0;
+
+  /* Take a mutable copy of outlen to count down the number of bytes still to
+   * squeeze. The initial (const) value of outlen is needed for the CBMC
```

Comment:
```suggestion
   * squeeze. The initial value of outlen is needed for the CBMC
```
Nit: adjust comment to reflect the removal of the `const`

Reply:
Done